examples/tfengine/generated/org_foundation/cicd/main.tf

```diff
@@ -176,7 +176,7 @@ resource "google_organization_iam_member" "cloudbuild_sa_organization_iam" {
 
 # Create Google Cloud Build triggers for specified environments
 module "triggers" {
-  for_each = var.envs
```

Comment:
It's a list of strings, you can use toset(var.envs) for this.

Reply:
No, it's a list of objects. The error that gives me when used as `var.envs` is this:
![Screenshot 2021-07-22 4 45 17 PM](https://user-images.githubusercontent.com/33379285/126713457-cc022537-0a47-4e3a-8c56-62aa0bc70356.png)

And with `{for env in var.envs:  env.name => env}` is this:

![Screenshot 2021-07-22 4 43 28 PM](https://user-images.githubusercontent.com/33379285/126713391-09b6a1a8-bd01-4d99-8203-84c9d8c22ab2.png)

Reply:
Got it. You're right. I thought it's a list of strings.